packages/angular_devkit/schematics/src/tree/action_spec.ts

```diff
@@ -87,7 +87,7 @@ describe('Action', () => {
       actions.optimize();
       expect(actions.length).toBe(1);
       expect(actions.get(0)).toEqual(
-        jasmine.objectContaining<Action>({ kind: 'c', path: normalize('/test2') }),
+        jasmine.objectContaining({ kind: 'c', path: normalize('/test2') }),
```

Comment:
casting the object literal should fix the errors

Reply:
:+1: